build_verify.sh

```diff
@@ -0,0 +1,77 @@
+#/usr/bin/env bash 
+
+
+fails=""
+
+# Invoke gofmt on the current directory.
+# -l flag causes it to only list files
+GOFMT_RESULTS=`gofmt -l .`
+if [ "$GOFMT_RESULTS" ]
+then
+	echo "FAIL: UNFORMATTED FILES:"
+	echo "GOFMT FINDS"
+	echo "$GOFMT_RESULTS"
+	fails="fails"
+fi
+
+
+# Find all files ending with .go that do not end with .peg.go (generated)
+GOLINT_FILES=`find . -not -name *.peg.go -name *.go`
+# Iterate over these (split by newline)
+
+# Check each of them with golint
```

Comment:
Not sure there is one. I kind of agree with the sentiment that things should either pass/fail and have nothing in between though. If it's something we really want to warn about, we should fail. Especially in the case of golint, we'd perpetually be in a warning state if we never fix it.

Maybe we can pull out the `golint` logic into its own script so we can run it in dev.